src/components/explorer/ExplorerSearchResultsTable.js

```diff
@@ -0,0 +1,180 @@
+import React, { useState, useEffect, useMemo } from "react";
+import PropTypes from "prop-types";
+import { useParams } from "react-router-dom";
+import { useSelector, useDispatch } from "react-redux";
+import { Table, Typography, Button, Spin } from "antd";
+
+import SearchSummaryModal from "./SearchSummaryModal";
+import SearchTracksModal from "./SearchTracksModal";
+
+import {
+    setSelectedRows,
+    performIndividualsDownloadCSVIfPossible,
+    performBiosamplesDownloadCSVIfPossible,
+    performExperimentsDownloadCSVIfPossible,
+} from "../../modules/explorer/actions";
+
+const ExplorerSearchResultsTable = ({ data, activeTab, ...props }) => {
+    const { dataset } = useParams();
+    const [currentPage, setCurrentPage] = useState(1);
+    const [pageSize] = useState(25);
```

Comment:
If we dont update the `pageSize` state we can move it to a const outside the component

Reply:
Perfect!